README.md

```diff
@@ -215,54 +213,48 @@ To create at <i>tray icon</i> launcher with a .desktop file, follow the instruci
+Nonetheless, if you would like to launch the GUI without having to type in your sudo password everytime, then you could add the executable file to `visudo`. This is extremly useful when you have a .desktop file, and all you want to do is click the launcher to have the GUI pop-up.<br>
+Another alternative is to launch the GUI with `pkexec`. What this does is that it displays a window for you to type the sudo password, this way, you won't have to use the terminal to type in the password. **Note:** This feature needs to be enabled from within the GUI settings for it to be able to work. If you have not installed the necessary packages then this feature will not be able to be activated.
+
+### Visudo
+The instructions for seting up visudo are described below.
 
 1. First you will need the path to the GUI. This can be found by typing `which protonvpn-gui`. You should get something like this: `/usr/bin/protonvpn-gui`. Save it since you will need it later. **Note:** As previously mentioned, the path may look different for you, based on your distro.
 2. Identify your username by typing `whoami`. Save it (or memorize it). 
 3. In another terminal, type in `sudo visudo`, and a window should pop-up, scroll to the very bottom of it.
 4. Once you are at the botton, type: `<YOUR_USERNAME_FROM_STEP2> ALL = (root) NOPASSWD: <YOUR_PATH_FROM_STEP1>`
 5. Exit and save! Have fun :)
 
-# Not yet implemented:
-
-- ~~Split Tunneling~~
-- ~~Kill Switch~~
-- ~~Filtering servers~~
-- ~~Start on Boot~~ (only for systemd/systemctl based OS's)
-- ~~Systray/AppIndicator~~
-
 # GUI Layout
 <p align="center">
-  <img src="https://i.imgur.com/OlcYrwf.png" alt="Login"></img>
-</p>
-<p align="center">
-  <img src="https://i.imgur.com/VOpTlMR.png" alt="Dashboard"></img>
+  <img src="https://i.imgur.com/ZmWlHV5.png" alt="Login"></img>
 </p>
 <p align="center">
-  <img src="https://i.imgur.com/BlVmFa9.png" alt="Profiles"></img>
+  <img src="https://i.imgur.com/aM9tTgd.png" alt="Dashboard"></img>
 </p>
 
 <p align="center">
-  <img src="https://i.imgur.com/S353btA.png" alt="General Settings"></img>
+  <img src="https://i.imgur.com/ScIcw10.png" alt="General Settings"></img>
 </p>
 <p align="center">
-  <img src="https://i.imgur.com/BWE9LDf.png" alt="Tray Settings"></img>
+  <img src="https://i.imgur.com/Nj0c5xf.png" alt="Tray Settings"></img>
 </p> 
 <p align="center">
-  <img src="https://i.imgur.com/md4ni6t.png" alt="Connection Settings"></img>
+  <img src="https://i.imgur.com/KMdDszk.png" alt="Connection Settings"></img>
 </p>  
 <p align="center">
-  <img src="https://i.imgur.com/KYMxSoe.png" alt="Advanced Settings"></img>
+  <img src="https://i.imgur.com/QkLbcuo.png" alt="Advanced Settings"></img>
```

Comment:
We could migrate all these picture into GitHub repo itself.

Reply:
Will do! :)

Reply:
I'm planning to have an "images/" folder on the top level of the CLI for this as well. Trying to do this later today, so maybe we can do this consistent. Or would you do it differently?

Reply:
@Rafficer I already have an "img/" folder in /resources". It would not make much sense to put in top level (in my case) I think, since I would end up with two different image folders.